server/services/placeOnReportService.ts

```diff
@@ -267,4 +274,26 @@ export default class PlaceOnReportService {
       })
     )
   }
+
+  async getPlaceholderValues(adjudicationNumber: number, user: User) {
```

Comment:
Not sure about the name, given it is for offence selection - maybe `getOffenceSelectionPlaceholderValues`?

Reply:
will change the name

Reply:
Done